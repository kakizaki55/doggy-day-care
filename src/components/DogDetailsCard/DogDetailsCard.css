.dog-details-card {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: 1rem;
  border: burlywood solid 5px;
  padding: 1rem;
}
