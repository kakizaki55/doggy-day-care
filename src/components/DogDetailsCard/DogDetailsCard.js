import React from 'react';
import './DogDetailsCard.css';
import { Link } from 'react-router-dom/cjs/react-router-dom.min';

export default function DogDetailsCard({ name, age, breed, image, bio }) {
  return (
    <div className="dog-details-card">
      <h1> Meet {name} ! </h1>
      <img src={`${image}`}></img>
      <p>
        {name} is a(n) {age} year old {breed}{' '}
      </p>
      <p>{bio}</p>
      <Link to="/">back to all the Doggos</Link>
    </div>
  );
}
