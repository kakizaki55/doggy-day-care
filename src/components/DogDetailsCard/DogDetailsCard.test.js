import DogDetailsCard from './DogDetailsCard';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

test('making sure the dog deatils card is reding correctly', async () => {
  const container = await render(
    <BrowserRouter>
      <DogDetailsCard
        name="tommy"
        breed="husky"
        bio="loves evveryone"
        image="https://placedog.net/500?id=6"
        age={3}
      />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
