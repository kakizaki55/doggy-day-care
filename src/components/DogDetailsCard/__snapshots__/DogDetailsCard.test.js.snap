// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`making sure the dog deatils card is reding correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dog-details-card"
      >
        <h1>
           Meet 
          tommy
           ! 
        </h1>
        <img
          src="https://placedog.net/500?id=6"
        />
        <p>
          tommy
           is a(n) 
          3
           year old 
          husky
           
        </p>
        <p>
          loves evveryone
        </p>
        <a
          href="/"
        >
          back to all the Doggos
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dog-details-card"
    >
      <h1>
         Meet 
        tommy
         ! 
      </h1>
      <img
        src="https://placedog.net/500?id=6"
      />
      <p>
        tommy
         is a(n) 
        3
         year old 
        husky
         
      </p>
      <p>
        loves evveryone
      </p>
      <a
        href="/"
      >
        back to all the Doggos
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
